utils/Addresses.json

```diff
@@ -299,6 +299,12 @@
         "name": "USDBC",
         "isContract": true
     },
+    {
+        "addr": "0x9e3380f8B29E8f85cA19EFFA80Fb41149417D943",
```

Comment:
Confirmed this is the correct URD provided by Morpho with owner [0x937Ce2d6c488b361825D2DB5e8A70e26d48afEd5](https://basescan.org/address/0x937Ce2d6c488b361825D2DB5e8A70e26d48afEd5)